expressions/aggregation/AggregationHandleCount.hpp

```diff
@@ -62,6 +63,10 @@ class AggregationStateCount : public AggregationState {
    */
   ~AggregationStateCount() override {}
 
+  size_t getPayloadSize() const {
```

Comment:
It is considered a good practice to use the ``std`` prefix in the header file. Therefore, can you please change the ``size_t`` to ``std::size_t``?